src/test/scala/io/iohk/ethereum/cli/CliCommandsSpec.scala

```diff
@@ -92,6 +94,30 @@ class CliCommandsSpec extends AnyFlatSpec with Matchers with EitherValues {
     val stringSplit = result.right.get.split("\\n\\n")
     stringSplit.length shouldEqual numOfKeysAsInt
   }
+
+  behavior of encryptKeyCommand
+  it should "encrypt private key (without passphrase)" in {
+    val json = api.parse(Seq(encryptKeyCommand, privateKey)).value
+
+    val decrypted = (for {
+      encrypted <- EncryptedKeyJsonCodec.fromJson(json)
+      decrypted <- encrypted.decrypt("")
+    } yield decrypted).value
+
+    ByteStringUtils.hash2string(decrypted) shouldBe privateKey
+  }
+
+  it should "encrypt private key (with passphrase)" in {
```

Comment:
minor: this test looks like a perfect candidate for a property-based one.

Reply:
Not really. I don't want to test decryption/encryption here. It should be done in a different place. I only want to check if I get the correct JSON when I specify the correct arguments